layouts/default.vue

```diff
@@ -12,6 +12,13 @@ import Footer from '../components/Footer.vue'
 
 export default {
   components: { Footer, Navbar },
+  computed: {
+    isTemplateDetailPage() {
+      return (
+        this.$route.path.startsWith('/templates/') && this.$route.params?.slug
+      )
+    },
+  },
```

Comment:
let's create a new layout, that's a better architecture and let's use that on templates page